modules/bucket-events/main.tf

```diff
@@ -0,0 +1,214 @@
+terraform {
+  required_providers {
+    ko     = { source = "ko-build/ko" }
+    cosign = { source = "chainguard-dev/cosign" }
+  }
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  upper   = false
+  special = false
+}
+
+// A dedicated service account for this subscription.
+resource "google_service_account" "this" {
+  project = var.project_id
+
+  account_id   = "${var.name}-${random_string.suffix.result}"
```

Comment:
I think that maybe this (and below) should just be `var.name` and the burden of unique naming should be on the caller.  WDYT?